src/DatePicker/__stories__/DatePicker.stories.js

```diff
@@ -10,6 +10,7 @@ import DatePickerReadme from '../README.md';
 
 const onChangeAction = action('onChange');
 const today = new Date();
+const debutOfToday = new Date(today.getFullYear(), today.getMonth(), today.getDate(), 0, 0, 0);
```

Comment:
`debutOfToday` sound weird does it possible to change the naming with something like `todayBeginning` ?